pkgs/development/compilers/picat/default.nix

```
@@ -1,27 +1,29 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, zlib, gcc7Stdenv }:
 
-stdenv.mkDerivation {
-  name = "picat-1.9-4";
+gcc7Stdenv.mkDerivation {
+  name = "picat-2.7b12";
 
   src = fetchurl {
-    url = http://picat-lang.org/download/picat19_src.tar.gz;
-    sha256 = "0wvl95gf4pjs93632g4wi0mw1glzzhjp9g4xg93ll2zxggbxibli";
+    url = http://picat-lang.org/download/picat27b12_src.tar.gz;
```

Allow for bot to update package
```suggestion
    url = http://picat-lang.org/download/picat${stdenv.lib.replaceStrings ["."] [""] version}_src.tar.gz;
```